README.md

```diff
@@ -22,28 +22,38 @@ You can use a ready-to-use ISO image from the [Releases](https://github.com/jose
 
     ~~~
     sudo apt update
-    sudo apt -y install git python3-pip python3-smbus python3-numpy libportaudio2 
+    sudo apt -y install git python3-pip python3-smbus python3-numpy libportaudio2 libasound2-dev
     sudo apt -y install raspberrypi-kernel  # quite long to install, do it only if necessary, it solves a "no sound before 25 second on boot" problem
-    sudo pip3 install cython rtmidi-python cffi sounddevice pyserial
+    sudo pip3 install cython cffi sounddevice pyserial
     ~~~
-    
-2. Download SamplerBox and build it with:
+
+    For python < 3.9
+    ~~~
+    sudo pip3 install rtmidi-python
+    ~~~
+
+    For python >= 3.9
+    ~~~
+    sudo pip3 install python-rtmidi
+    ~~~
+
+1. Download SamplerBox and build it with:
```

Comment:
Just nitpicking about the details. But while the result/render is probably the same, I don't see the benefit of replacing `2.` with `1.` here.

Reply:
Thanks for your feedback.

using `1.` in markdown lists delegates the enumeration to the rendering. So if you have a list and want to add some item in the middle you do not have to rename all the other items.  

![image](https://github.com/josephernest/SamplerBox/assets/2125774/85fbd4c6-234f-4fff-9670-acb14a1a7e19)